nni/contrib/compression/utils/evaluator.py

```diff
@@ -216,13 +216,13 @@ def add_param(param_lis: List[Tensor], target_param_group_idx: int, optimizer: O
                 # copyed from torch.optim to check the validation of param
                 if not isinstance(param, torch.Tensor):
                     raise TypeError("optimizer can only optimize Tensors, "
-                                "but one of the params is " + torch.typename(param))
+                                    "but one of the params is " + torch.typename(param))
                 if not optimizer.defaults.get('differentiable', None) \
-                    and not (param.is_leaf or param.retains_grad): # type: ignore
+                    and not (param.is_leaf or param.retains_grad):  # type: ignore
                     raise ValueError("can't optimize a non-leaf Tensor")
                 target_param_group['params'].append(param)
 
-        assert isinstance(model, (Module, pl.LightningModule))
+        assert isinstance(model, Module)
```

Comment:
what if model is pl.LightingModule?

Reply:
`pl.LightingModule` is a subclass of `torch.nn.Module`, here remove it because lightning is not a required dependency for using compression.